src/DashboardService.Mappers/Models/ChangeLogInfoMapper.cs

```diff
@@ -0,0 +1,32 @@
+﻿using LT.DigitalOffice.DashboardService.Mappers.Models.Intarfaces;
+using LT.DigitalOffice.DashboardService.Models.Db;
+using LT.DigitalOffice.DashboardService.Models.Dto.Models;
+
+namespace LT.DigitalOffice.DashboardService.Mappers.Models;
+
+public class ChangeLogInfoMapper : IChangeLogInfoMapper
+{
+  //private readonly ITaskInfoMapper _taskInfoMapper;
+
+  public ChangeLogInfoMapper(/*ITaskInfoMapper taskInfoMapper*/)
+  {
+    //_taskInfoMapper = taskInfoMapper;
+  }
+
+  public ChangeLogInfo Map(DbChangeLog dbChangeLog)
+  {
+    return dbChangeLog is null ?
```

Comment:
? and : on next line. Like in this PR https://github.com/LT-Students/DashboardService/pull/10/files

Reply:
+

Reply:
null check is redundant actually even though you haven't fixed styling